autodistill/detection/detection_base_model.py

```diff
@@ -56,5 +64,16 @@ def label(
 
         split_data(output_folder)
 
+        if (
+            human_in_the_loop
+            and roboflow_api_key is not None
+            and roboflow_workspace is not None
+            and roboflow_project is not None
+        ):
+            rf = Roboflow(api_key=roboflow_api_key)
+            workspace = rf.workspace(roboflow_workspace)
+
+            workspace.upload_dataset(output_folder, project_name=roboflow_project)
```

Comment:
This should add a `autodistill` tag so you can filter by ones autolabeled. We should also allow people to add additional tags to uploaded images.

Can it add them to a specific job to keep them organized better?

Reply:
Good call! I'll check the `roboflow` Python package and see.

Reply:
Bump on this question

Reply:
This is not supported.

https://github.com/roboflow/roboflow-python/blob/ed840653ef540366c3e9e28bc429037ca20dc571/roboflow/core/workspace.py#L291